### PR TITLE
Add: Player Example

### PR DESCRIPTION
**Pull request by Strokkur424**
Adds an example to the safe suggestions to show a more 'daily' use of the sage suggestions:
The example consists of a /warp <player> command, where the player suggests only suggest player names, and no entity selector suggestions, as one would expect.

### CODE REVIEW

**Comment by Strokkur424** [comment]
I was thinking about putting this into the player page too, but I ultimately put it here, as it shows a great usecase for the safesuggestions. I can put it into the PlayerArgument page though

**Comment by Strokkur424** [comment]
Mainly because the player argument page is not specific to players, but to EntitySelectors in general

**Comment by DerEchtePilz** [comment]
Well, there's a section about the `PlayerArgument`, I think you could put it there.

File: commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt [hunk 1]
```diff
@@ -303,7 +303,7 @@ commandAPICommand("remove") {
 }
 /* ANCHOR_END: argumentEntities1 */
 
-/* ANCHOR: argumentEntities2 */
+/* ANCHOR: argumentEntities4 */
```
Comment (DerEchtePilz) [inline commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt:306]: There aren't actually four Kotlin DSL examples, please undo this change.

File: docssrc/src/argument_entities.md [hunk 2]
```diff
@@ -102,15 +145,15 @@ Since we're trying to specify an entity type, we will use the `EntityTypeArgumen
...
 ```
 
 ```kotlin,Kotlin
-{{#include ../../commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/Examples.kt:argumentEntities2}}
+{{#include ../../commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/Examples.kt:argumentEntities4}}
 ```
 
 ```kotlin,Kotlin_DSL
-{{#include ../../commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt:argumentEntities2}}
+{{#include ../../commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt:argumentEntities4}}
```
Comment (DerEchtePilz) [inline docssrc/src/argument_entities.md:156]: Lastly here, we don't have four Kotlin DSL examples.

File: docssrc/src/argument_entities.md [hunk 3]
```diff
@@ -71,6 +71,49 @@ The `PlayerArgument` class is very similar _(almost identical)_ to `EntitySelect
 >
 > The `PlayerArgument` internally uses the `GameProfile` class from Mojang's authlib, which means that this argument has a slight performance overhead compared to using `EntitySelectorArgument.OnePlayer`
 
+<div class="example">
+
+### Example - Warp command
```
Comment (DerEchtePilz) [inline docssrc/src/argument_entities.md:76]: Hmm, not sure about the example title. Maybe something like "`PlayerArgument` without entity selectors" would be better?

File: docssrc/src/argument_entities.md [hunk 4]
```diff
@@ -71,6 +71,49 @@ The `PlayerArgument` class is very similar _(almost identical)_ to `EntitySelect
...
+
+When registering a `PlayerArgument` you might notice that it includes `Entity Selectors` (`@a`, `@e`, `@r`, etc.). If you want to avoid those, you can use safe argument suggestions to only suggest the player names. For this example, let us create a /warp command:
```
Comment (DerEchtePilz) [inline docssrc/src/argument_entities.md:78]:
```suggestion
When registering a `PlayerArgument` you might notice that it includes `Entity Selectors` (`@a`, `@e`, `@r`, etc.). If you want to avoid those, you can use argument suggestions to only suggest the player names. For this example, let us create a /warp command:
```

As this is not specific to safe suggestions, it might be nicer to phrase it this way.

**Review by DerEchtePilz — CHANGES_REQUESTED** [review]
Some minor points, otherwise looks good :+1:

**Review by willkroboth — COMMENTED** [review]
It might be nice to add a quick note that while the argument will not suggest selectors, they will still be accepted to the argument as usual. Just to avoid any possible confusion if developers use this and expect that selectors would no longer be accepted.

File: docssrc/src/argument_entities.md (hunk 4 above)
Comment (Strokkur424) [reply docssrc/src/argument_entities.md]: Changed it

File: docssrc/src/argument_entities.md (hunk 3 above)
Comment (Strokkur424) [reply docssrc/src/argument_entities.md]: Changed 🎉

File: docssrc/src/argument_entities.md (hunk 2 above)
Comment (Strokkur424) [reply docssrc/src/argument_entities.md]: Reverted the change

File: commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt [hunk 5]
```diff
@@ -313,7 +313,7 @@ commandAPICommand("spawnmob") {
         }
     }
 }
-/* ANCHOR_END: argumentEntities2 */
+/* ANCHOR_END: argumentEntities4 */
```
[the remark this answers is not in the document]
Comment (Strokkur424) [reply commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt]: Reverted

File: commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt (hunk 1 above)
Comment (Strokkur424) [reply commandapi-documentation-code/src/main/kotlin/dev/jorel/commandapi/examples/kotlin/ExamplesKotlinDSL.kt]: Revered 🎉

**Comment by Strokkur424** [comment]
> It might be nice to add a quick note that while the argument will not suggest selectors, they will still be accepted to the argument as usual. Just to avoid any possible confusion if developers use this and expect that selectors would no longer be accepted.

Added a note right before the gif. If you want me to rephrase, I would be glad to!

**Review by DerEchtePilz — APPROVED** [review]

**Review by willkroboth — APPROVED** [review]
